app/templates/common/_nav.html

```diff
@@ -1,98 +1,100 @@
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto">
+                    <li class="nav-item display-lg display-md">
+                        {% if request.endpoint == 'index_routes.index' %}
+                            <a class="nav-link" aria-current="page" href="/">
+                                <i class="bi bi-house-door-fill"></i>
+                            </a>
+                        {% else %}
+                            <a class="nav-link" aria-current="page" href="/">
+                                <i class="bi bi-house-door"></i>
+                            </a>
+                        {% endif %}
+                        <li class="nav-item display-lg display-md">
+                            <a class="nav-link" aria-current="page" href="/post">
+                                <i class="bi bi-plus-square"></i>
+                            </a>
+                        
+                        
+                    </li>
+                    <li class="nav-item dropdown display-lg display-md">
+                        <a class="nav-link" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown">
+                            <img src="{{ url_for('static', filename='assets/default-profile-picture.png') }}" class="navbar-profile-pic rounded-circle">
+                        </a>
+                        <ul class="dropdown-menu dropdown-menu-end border-0 shadow rounded" aria-labelledby="navbarDropdown">
+                            <span class="dropdown-menu-arrow"></span>
+                            <li>
+                                <a class="dropdown-item" href="/{{ session.get('logged_in').get('username') }}">
+                                    <i class="bi bi-person-circle align-top"></i>
+                                    <span class="align-middle">Profile</span>
+                                </a>
+                            </li>
+{# TODO: Uncomment when such features are added #}
+{#                            <li>#}
+{#                                <a class="dropdown-item" href="#">#}
+{#                                    <i class="bi bi-gear-wide align-top"></i>#}
+{#                                    <span class="align-middle">Settings</span>#}
+{#                                </a>#}
+{#                            </li>#}
+{#                            <li>#}
+{#                                <hr class="dropdown-divider">#}
+{#                            </li>#}
+                            <li>
+                                <form method=post action=account/signout>
+                                    <button class="dropdown-item">

```

Comment:
Remove the button click css.

```
button.dropdown-item:focus {
outline: 0;
}
```

Should do it.

Reply:
fixed in latest commit